docs/ml-guide.md

```diff
@@ -801,6 +801,146 @@ jsc.stop();
 
 </div>
 
+## Examples: Summaries for LogisticRegression.
+
+Once [`LogisticRegression`](api/scala/index.html#org.apache.spark.ml.classification.LogisticRegression)
+is run on data, it is useful to extract statistics such as the
+loss per iteration which will provide an intuition on overfitting and metrics to understand
+how well the model has performed on training and test data.
+
+[`LogisticRegressionTrainingSummary`](api/scala/index.html#org.apache.spark.mllib.classification.LogisticRegressionTrainingSummary)
+provides an interface to access such relevant information. i.e the objectiveHistory and metrics
+to evaluate the performance on the training data directly with very less code to be rewritten by
+the user. In the future, a method would be made available in the fitted
```

Comment:
nit: I would just leave this part of the doc out in case we forget to remove it when we do settle on a good name for `evaluate`.